src/meta/src/model/stream.rs

```diff
@@ -264,25 +284,6 @@ impl TableFragments {
         .collect()
     }
 
-    /// Returns fragments that contains Chain node.
-    pub fn chain_fragment_ids(&self) -> HashSet<FragmentId> {
```

Comment:
A lot of code in this file can be removed due to no more need for `resolve_chain`.